automation_oca/README.rst

```diff
@@ -0,0 +1,169 @@
+==============
+Automation Oca
+==============
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:5d9710e6bc1697d84bcd87bdd88f0cf9544b2f82053cf7dc3f5c232f788a4a07
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fautomation-lightgray.png?logo=github
+    :target: https://github.com/OCA/automation/tree/16.0/automation_oca
+    :alt: OCA/automation
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/automation-16-0/automation-16-0-automation_oca
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/automation&target_branch=16.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module allows to automate several process according to some rules.
+
+This is useful for creating automated actions on your database like:
+
+-  Send a welcome email to all new partners (or filtered according to
+   some rules)
+-  Remember to online customers that they forgot their basket with some
+   items
+-  Send documents to sign to all new employees
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+Configure your processes
+------------------------
+
+1. Access the ``Automation`` menu.
+2. Create a new Configuration.
+3. Set the model and filters.
+4. Create the different activities.
```

Comment:
to create activities the user should press te add button inside the automation configuration form